sentences/fr/_common.yaml

```diff
@@ -493,12 +493,12 @@ expansion_rules:
 
   # Verbs
   active: "(active|activer|joue|jouer|exécute|exécuter|démarre|démarrer|lance|lancer)"
-  allume: "(allume|allumer|active|activer|démarre|démarrer)"
+  allume: "(allume|allumer|active|activer|démarre|démarrer|ouvre|ouvrir)"
   augmente: "(augmente|augmenter|monte|monter)"
   demarre: "(démarre|démarrer|lance|lancer)"
   diminue: "(diminue|diminuer|baisse|baisser)"
   eclaire: "(éclaire|éclairer|illumine|illuminer)"
-  eteins: "(éteint|eteint|éteins|eteins|éteindre|eteindre|désactive|désactiver|stoppe|stopper|arrête|arrêter|coupe|couper|<eteins_dirty>)"
+  eteins: "(éteint|eteint|éteins|eteins|éteindre|eteindre|désactive|désactiver|stoppe|stopper|arrête|arrêter|coupe|couper|ferme|fermer|<eteins_dirty>)"
```

Comment:
I added `ferme` and `fermer` here instead of `<ferme>`, because the definition of `<ferme>` includes <baisse>.